src/components/ListWidgetItem.js

```diff
@@ -1,12 +1,18 @@
 var React = require('react');
 
 var ListWidgetItem = React.createClass({
+    renderTags: function() {
+        return (this.props.tags || []).map(function(name) {
+            return(<span key={Math.random()} class="badge badge-secondary">{name}</span>)
```

Comment:
@footnoise , I am curious on why you put `key={Math.random()}`

Reply:
@krazedkrish `key` should be unique number, if we using like `{idx}`(just index of element in array) in that case we can get warning that similar object with the the key is presented in DOM.
`Math.random()` is pretty safe it generates you a good unique numbers.